src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -148,9 +145,8 @@ public void ConfigureServices(IServiceCollection services)
 #endif
 
             services.AddHealthChecks();
-#if ExcludeCorrelation
-#else
-            services.AddCorrelation();
+#if ExcludeCorrelation == false
+            services.AddHttpCorrelation();
```

Comment:
This change is actually a a "fix" for the correlation.

Reply:
Thanks!